wiki-config

```diff
@@ -1,16 +1,38 @@
-# wiki-config
+#!/bin/bash
+
+#set -x
+set -e
+set -o pipefail
+set -o nounset
+
+## TODO:
+## create a file:
+## credentials
+## with the following contents:
+#[[ -v USERNAME ]] || USERNAME="username"
+#[[ -v USERPASS ]] || USERPASS="password"
 
 # Location of wiki's api.php
-WIKIAPI="https://wiki.example/Wiki/api.php"
+[[ -v WIKIAPI ]] || WIKIAPI="https://www.whonix.org/w/api.php"
```

Comment:
`wiki.example` not advertising
this is a config file, it shouldn't be testing for variables
maybe instead do something like `[ ! -e wiki-config ] || . wiki-config` in the scripts so the config can be skipped and variables will instead be pulled from the env